src/test/java/com/google/api/codegen/protoannotations/testdata/library.proto

```diff
@@ -354,7 +443,7 @@ message BookFromArchive {
   // Book names have the form `archives/{archive_id}/books/{book_id}`.
   string name = 1 [
     (google.api.required) = true,
-    (google.api.resource_type) = "google.example.library.v1.Book"];
+    (google.api.resource_type) = "google.example.library.v1.Book.archived_book"];
```

Comment:
Is this how you specify a Resource in a ResourceSet? where `archived_book` is a resource.base_name in the ResourceSet `book_oneof`.